csp-guru/csp-guru.csv

```diff
@@ -1,110 +1,112 @@
-Abijheet,,construction,India,Phalodi,26.8278,70.9197,1985,2013,Trough,50.0,0.0,,Wet,,,,,,Corporate Ispat Alloys,Siemens,Ener-t Internationa,PPA,12.24 INR/kWh,0.1909,0.1720,,,,,,Corporate Ispat Alloys,,,254,http://www.nrel.gov/csp/solarpaces/project_detail.cfm/projectID=254
-Diwakar,,construction,India,Askandra,27.3571,71.6918,1938,2013,Trough,100.0,4.0,,Wet,,,,,,Lanco Solar,,Sener,PPA,10.05 INR/kWh,0.1568,0.1413,,,,,,Lanco Solar,,,258,http://www.nrel.gov/csp/solarpaces/project_detail.cfm/projectID=258; SUNDERASAN 2015
-KVK,,construction,India,Askandra,27.3819,71.7731,1940,,Trough,100.0,4.0,,Wet,,,,,,KVK,,Sener,PPA,11.1 INR/kWh,0.1731,0.1560,,,,,,KVK,,,260,http://www.nrel.gov/csp/solarpaces/project_detail.cfm/projectID=260; SUNDERASAN 2015
-Agua Prieta II,,construction,Mexico,Agua Prieta,31.3255,-109.5471,2799,2011,Trough,12.0,0.0,85000,,34.0000,0.3234,,,,Abengoa,,Abengoa,,,,,,,,,Global environmental facility,Federal Electricity Commission,,,135,http://www.nrel.gov/csp/solarpaces/project_detail.cfm/projectID=135
-Stillwater GeoSolar,,construction,USA,Fallon,39.5429,-118.5619,2558,2014,Hybrid; Trough,17.0,0.0,,,3.0000,0.0201,,,,Enel Green Power,,SkyFuel ,PPA,,,,,,,,,Enel Green Power,20.0000,,4279,http://www.nrel.gov/csp/solarpaces/project_detail.cfm/projectID=4279
+Power_station,Year_operational,Status,Country,City,Lat,Lon,DNI_[kWh/m2/yr],Construction_start,Technology,Capacity_[MW],Storage_capacity_[h],Field_size_[m2],Cooling,Expected_generation_[GWh/yr],Load_factor,Total_cost_[million_currency;non-deflated],Total_cost_[million_USD;non-deflated],Total_cost_[million_USD2017;deflated],Specific_investment_cost_[USD2017/kW;deflated],Total_cost_[million_EURO;non-deflated],Total_cost_[million_EURO2017;deflated],Specific_investment_cost_[EURO2017/kW;deflated],Developer,EPC,HCE_manufacturer,SCA_manufacturer,Financed_through,Remuneration_[currency/kWh],Remuneration_[USD/kWh;non_deflated],Remuneration_[USD2017/kWh;deflated],Remuneration_[EURO/kWh;non_deflated],Remuneration_[EURO2017/kWh;deflated],LCOE_5_25_[USD2017],LCOE_10_25_[USD2017],LCOE_5_25_[EURO2017],LCOE_10_25_[EURO2017],Concessional_funding;other_support,Owner,PPA_or_support_duration_[yr],Sources
+SEGS I,1984,operating,USA,Daggett,34.86305556,-116.8269444,2885,#N/A,Trough,13.8,3,82960,Wet,16.5,0.136489974,62.1,62.1,127.1224168,9211.769336,79.26141803,151.3250749,10965.58514,Luz,#N/A,#N/A,Luz,PPA,#N/A,#N/A,#N/A,#N/A,#N/A,0.662211158,0.964343506,0.788288609,1.147943509,PURPA; Accelerated depreciation; Federal and state investment incentives; Solar property tax exclusion,Cogentrix,#N/A,Lillestam et al 2017: Nature energy
```

Comment:
I propose to replace the newly introduced `,#N/A,` with what we had before: an empty field like `,,`. This lets less room for misinterpretation.

Reply:
changed.